### PR TITLE
Changes in docs regarding the new is_highlighted field on dashboards

### PR DESCRIPTION
**Pull request by ikas**
Added documentation regarding the field `is_highlighted`, namely:
* Filters by this field
* How/who can provide this field when creating/editing this field

Related PT task: https://www.pivotaltracker.com/n/projects/1883443/stories/169809569

### CODE REVIEW

File: source/includes/api/_dashboard.md [hunk 1]
```diff
@@ -148,23 +150,25 @@ curl -X GET https://api.resourcewatch.org/v1/dashboard?includes=user
 
 When creating a dashboard, the `application` field should be present and cannot contain any values that are not associated with the creating user's account. If an `application` value is not provided, `["rw"]` is used by default, and the process will fail if the user account does not belong to it. Any role can create a dashboard.
 
-Supported fields:
+**The field `is-highlighted` can only be provided on creation by ADMIN users.** If you provide this field and the token does not match an ADMIN user, the request will fail with status code **403 Forbidden** and the message `You need to be an ADMIN to create/update the is_highlighted attribute of the dashboard.`
```
Comment (tiagojsag) [inline source/includes/api/_dashboard.md]: add this to the "description" below, and shorten it. if we document permissions for every field this way, it becomes unmanageable

**Review by tiagojsag — CHANGES_REQUESTED** [review]

File: source/includes/api/_dashboard.md [hunk 2]
```diff
@@ -59,6 +60,7 @@ private   |   Filter dashboards by private status (true, false)          | Boole
 user      |           Filter dashboards by author user id                | Text
 user.role | The role of the user who created the dashboard. If the requesting user does not have the ADMIN role, this filter is ignored. | `ADMIN`, `MANAGER` or `USER`
 application | The application to which the dashboard belongs             | Text (single value)
+isHighlighted | Filter dashboards by highlighted ones (true,false)       | Boolean
```
Comment (tiagojsag) [inline source/includes/api/_dashboard.md]: shoudn't this be dash case?

**Review by tiagojsag — CHANGES_REQUESTED** [review]

File: source/includes/api/_dashboard.md (hunk 2 above)
Comment (ikas) [reply source/includes/api/_dashboard.md]: Yes, it should, fixed 👍

File: source/includes/api/_dashboard.md (hunk 1 above)
Comment (ikas) [reply source/includes/api/_dashboard.md]: Done!

**Pull request merged**